README.md

```diff
@@ -57,6 +57,15 @@ Key tab for command completion, up/down for command history.
 The `goto` command takes you to a full screen 3270 interface. When
 space is available in the terminal screen/window, a status line is
 provided as well as a session selector.
+
+The basic syntax of the initial `goto` command is: 
+
+```
+goto fully-qualified-host-name [port]
+```
+
+If no port is specified, port 992, signifing to use TLS, is used.
```

Comment:
Well... 992 will be the default if using the default `SESSION_SSL=0`. Setting `SESSION_SSL=1` will make the default port 23. Maybe stating this is too complicated for the README?